### DISCUSSION TITLE
Foreign keys & indexing

### DISCUSSION THREAD

**Discussion by SpeedoPasanen** [noam-honig/radweb#192] [Ideas]
I see no way to set up foreign keys (with cascade) or indexes, besides writing migrations elsewhere. Is this on the roadmap?

**Comment by yoni-rapoport** [comment]
Hi @SpeedoPasanen, this is currently not on the roadmap because we're trying to focus on features that address full-stack code duplication and boilerplate plumbing code, rather than competing feature-to-feature with ORMs.

**Comment by SpeedoPasanen** [comment]
Good, focus on your strong points first.👍 If/when I have time, I'll look if I can do it by extending the Field decorator or adding extra decorators (possibly an extension npm package) and report back what I find.

Thanks ❤️🙏

**Reply by yoni-rapoport** [reply]
If you can - that would be awesome!
